.vib/appsmith/cypress/cypress/integration/appsmith_spec.js

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
+import {
+  random,
+} from '../support/utils';
+
+it('allows to create a new project', () => {
+  cy.login();
+  cy.get('button[class*="createnew"]').click();
```

Comment:
In most of these selectors, we could avoid setting the element used. It doesn't change that much but it reduces flakiness a bit. In this case, we would have the same with:

```suggestion
  cy.get('[class*="createnew"]').click();
```